.rubocop.yml

```diff
@@ -103,3 +103,6 @@ Security/YAMLLoad:
 
 Style/RegexpLiteral:
   Enabled: false
+
+Naming/VariableNumber:
```

Comment:
Disabling this rule as it appears to apply to keys returned from the bambora API.